### ISSUE TITLE
Error "compile" task - GRUNT

### ISSUE THREAD

**Issue by nhduy1985** [Xceptance/SiteGenesis-Community-TestSuite#84]
Error messages

```
D:\Demandware\Tools\xceptance>grunt
Running "all" task

Running "javaversion" task
Vers= 1.8.0_66

Running "checkPrerequisites" task

Running "compile" task
src\tests\others\TCheckoutOrder_AsGuest_RemoteIE.java:34: error: cannot find symbol
public class TCheckoutOrder_AsGuest_RemoteIE extends RemoteIETestRunner
                                                     ^
  symbol: class RemoteIETestRunner
1 error
Warning: Command failed: javac -classpath "..\xlt-4.6.1\lib\*" -d classes @sources.txt
src\tests\others\TCheckoutOrder_AsGuest_RemoteIE.java:34: error: cannot find symbol
public class TCheckoutOrder_AsGuest_RemoteIE extends RemoteIETestRunner
                                                     ^


  symbol: class RemoteIETestRunner
1 error
 Use --force to continue.

Aborted due to warnings.
```
Environments:
```
Windows 7
node v4.1.2
grunt-cli v0.1.13
grunt v0.4.5
```

It seems that RemoteIETestRunner is missing on sources.txt

[sources.txt](https://github.com/Xceptance/SiteGenesis-Community-TestSuite/files/142353/sources.txt)

**Comment by occupant23** [comment]
Hi Dustin,
Thank you for reporting this issue. I found this some time ago and have a fix ready that will be released soon.
In the mean time you can simply change the XLT options in the gruntfile.js to:

`
var xltOptions = {
pathToXLT: '../PATH_TO_XLT',
testCasesJava: '**/*.java', 
testCasesClass: '**/T*.class'
}; 
`

This will include RemoteIETestRunner to sources.txt.

**Comment by nhduy1985** [comment]
Thanks @occupant23 

It works like a charm!!!

**Comment by occupant23** [comment]
Fixed with pull request #88.
